MHTimer.js

```diff
@@ -2484,13 +2554,28 @@ function findRH(channel) {
                 }
                 responseStr = `Relic Hunter has been spotted in **${decode(body["rh"]["location"])}** and will be there for the next `;
                 responseStr += timeLeft(DateTime.utc().endOf('day'));
+                // NOTE: setting a higher scope variable inside this async code means it's not immediately set
+                relic_hunter.source = 'MHCT';
+                console.log(`Relic hunter source updated to MHCT`);
+                if (relic_hunter.location != decode(body["rh"]["location"])) {
+                    console.log(`Relic hunter location updated from ${relic_hunter.location} to ${decode(body["rh"]["location"])}`);
+                    relic_hunter.location = decode(body["rh"]["location"]);
+                    setImmediate(remindRH, relic_hunter.location);
+                    // Schedule a timer to do relic hunter reminders here IF the location changed
+                }
                 channel.send(responseStr);
-            }
-            else {
+            } else {
                 reportRequestError(`RelicHunter query failed:`, error, response, body);
                 channel.send("I was unable to get a good answer on that one.");
             }
         });
+    }
+    else {
+        // Assume we have a correct location then
```

Comment:
One way around this assumption is to store when the RH was seen, and then compute if we're on the same day as that report. If so, hooray, otherwise, requery or report unknown.

Reply:
Once upon a time I did keep track of which UTC day the RH location was last updated. Now it should reset at midnight to "unknown" which will always trigger an attempt to update the location. If it fails then people will now either see the old location (DBGames' update is based on when the #timers program runs) or 'unknown'.